agent/connect_ca_endpoint.go

```diff
@@ -15,13 +16,39 @@ func (s *HTTPHandlers) ConnectCARoots(resp http.ResponseWriter, req *http.Reques
 		return nil, nil
 	}
 
+	pemResponse := false
+	if pemParam := req.URL.Query().Get("pem"); pemParam != "" {
+		val, err := strconv.ParseBool(pemParam)
+		if err != nil {
+			return nil, BadRequestError{Reason: "The 'pem' query paramter must be a boolean value"}
```

Comment:
```suggestion
			return nil, BadRequestError{Reason: "The 'pem' query parameter must be a boolean value"}
```

Reply:
My spelling was apparently even more terrible this week than most.